index.html

```diff
@@ -33,9 +36,149 @@
+            <li class="label-project">HTML</li>    
+           
+        </ul>
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fourth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+          
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+           
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item fiveth"><div class="item image"></div> 
+        <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+       
+            <ul class="tagcontainer">
+         <li class="label-project">Ruby on Rails</li>
+         <li class="label-project">Javascript</li>
+         <li class="label-project">CSS</li>
+         <li class="label-project">HTML</li>
+            </ul>
+        
+        <button class="see-post"><p class="see-button">see project</p></button></div>
+        <div class="grid-item sixth">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+          
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+        <div class="grid-item ">
+            <div class="item image"></div> 
+           <h2 class="title">Multi-Post Stories Gain + Glory</h2>
+           
+            <ul class="tagcontainer">
+            <li class="label-project">Ruby on Rails</li>
+            <li class="label-project">Javascript</li>
+            <li class="label-project">CSS</li>
+            <li class="label-project">HTML</li>
+            </ul>
+           
+           <button class="see-post"><p class="see-button">see project</p></button>
+        </div>
+    </div>
+</section>
+<section>
+   
+        <div class="about-me">
+            
+              <!-- <img class="right" src="./right.png" alt="image">   -->
+            <h2 class="about">About Me</h2>
+            
+            <p class="about-myself"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+               
+                <button class="get-my-resume"><p class="resume">Get My Resume</p></button>
+                <!-- <img class="left" src="./left.png" alt="image"> -->
```

Comment:
- [ ] Leaving unused code as a comment in your codebase is against the best practices, :+1: I'll suggest you remove this as it makes your code base look unprofessional. :smile: